workers/email/scripts/emailOverview.ts

```diff
@@ -323,6 +329,48 @@ class EmailTestServer {
     return connection.collection('workspaces').findOne({ _id: new ObjectId(workspaceId) });
   }
 
+  /**
+   * Calculate days after payday
+   * Return number of days after payday. If payday is in the future, return 0
+   *
+   * @param workspace - workspace data
+   * @returns {Promise<number>} number of days after payday
+   */
+  private async calculateDaysAfterPayday(
+    workspace: WorkspaceDBScheme
+  ): Promise<number> {
+    /**
+     * Calculate number of days after payday
+     * The expected payday is either paidUntil or lastChargeDate + 1 month
+     * This follows the same logic as PaymasterWorker
+     */
+    let expectedPayDay: Date | null = null;
+
+    if (workspace.paidUntil) {
+      expectedPayDay = new Date(workspace.paidUntil);
+    } else if (workspace.lastChargeDate) {
+      const lastCharge = new Date(workspace.lastChargeDate);
+
+      expectedPayDay = new Date(lastCharge.getFullYear(), lastCharge.getMonth() + 1, lastCharge.getDate());
+    }
+
+    if (!expectedPayDay) {
+      return 0;
+    }
+
+    const now = new Date();
+    const diffTime = now.getTime() - expectedPayDay.getTime();
+
+    if (diffTime <= 0) {
+      return 0;
+    }
+
+    // Calculate difference in days
```

Comment:
I don't like copying of business logic in test script. It will became inconsistent if we change the original logic. 

I'd suggest to move the `daysAfterPayday` (along with the `daysBeforePayday`) methods from `workers/paymaster/index.ts` to `lib/utils/payday.ts` and reuse it as utility.